File: worker/worker-impl/src/main/java/com/masterello/worker/domain/TranslationLanguage.java
package com.masterello.worker.domain;

import lombok.Getter;

@Getter
public enum TranslationLanguage {
    DE("German"),  // German
    EN("English"),  // English
    UK("Ukrainian") ,  // Ukrainian
    RU("Russian"),  // Russian
    OTHER("other");

    private final String name;

    TranslationLanguage(String name) {
        this.name = name;
    }
}

File: translation/translation-contract/src/main/kotlin/com/masterello/translation/dto/TranslationLanguage.kt
package com.masterello.translation.dto

enum class TranslationLanguage(val sourceCode: String, val targetCode: String) {

    EN("en","en-US"),
    DE("de", "de"),
    RU("ru", "ru"),
    UK("uk", "uk"),
    OTHER("other", "other");
}